sway/tree/view.c

```diff
@@ -1171,7 +1171,7 @@ void view_update_title(struct sway_view *view, bool force) {
 
 	ipc_event_window(view->container, "title");
 
-	if (view->foreign_toplevel) {
+	if (view->foreign_toplevel && title != NULL) {
```

Comment:
Why mix 2 different styles of `NULL` check?
```suggestion
	if (view->foreign_toplevel && title) {
```

Reply:
Elsewhere in the file `title` is more common than `title != NULL`:
```
sway/tree/view.c
1141:           if (title && view->container->title &&
1145:           if (!title && !view->container->title) {
1152:   if (title) {
```

Reply:
Sway code in general is not super consistent on this (I suspect to avoid bikeshedding given there's no linter), but I agree it would be nice to keep things consistent within the same statement. @emersion, up to you, the PR looks functionally correct to me :)

Reply:
In general we don't have a preferred NULL check style in Sway/wlroots. However I agree that mixing two styles in the same condition is too inconsistent, applied your suggestion.